### PR TITLE
Enhanced German Translation

### LINKED ISSUE #31: Add German translations
For more info on translating, see the file `docs/Translating.md`

### PR DESCRIPTION
**Pull request by tintinmaster**
@ghizzo01 did already a good job at translating the website into german in #49 , but some sentences a native German would say in other ways or were just wrong. I fixed this now, so this PR should close #31

### CODE REVIEW

File: resources/lang/de/friends.php [hunk 1]
```diff
@@ -0,0 +1,67 @@
...
+        'title' => 'Hypixel Freundesliste',
+
+        'callout'             => [
+            "Zeigt eine Liste Ihrer Freunde oder einer anderen Person auf Hypixel an, wie lange du schon mit ihnen befreundet bist und weiteres",
+            "Ganz einfach – Gib einfach einen Benutzernamen ein um loszulegen."
+        ],
+        'callout_placeholder' => 'Geben Sie Ihren Benutzernamen um zu starten',
+        'callout_button'      => 'Freunde anzeigen',
+
+        'recently_viewed'                   => 'Zuletzt angesehene',
```
Comment (MaxKorlaar) [inline resources/lang/de/friends.php:43]: The other pages state 'angesehen', wouldn't that be the correct translation in this case?

File: resources/lang/de/friends.php [hunk 2]
```diff
@@ -0,0 +1,67 @@
...
+        'list' => [
+            'page_title' => "Hypixel-Freundesliste von :username",
+            'title'      => "Freunde von :username auf Hypixel",
+
+            'social' => [
+                'title'       => "Freunde von :username auf Hypixel - :site",
+                'description' => 'Sehen Sie sich alle Freunde von :total_friends Freunde von :username auf dem Hypixel Minecraft-Server an. Sie können durch die Liste blättern und zu anderen Hypixel-Spielern navigieren.',
+            ],
+            
+            // What is social doing?
```
Comment (MaxKorlaar) [inline resources/lang/de/friends.php:55]: The social translations are used in the meta tags, used for generating quick preview on social media sites:
https://github.com/MaxKorlaar/hypixel-signatures/blob/master/resources/views/partials/page.twig#L12-L23

File: resources/lang/de/home.php [hunk 3]
```diff
@@ -0,0 +1,63 @@
...
+        ],
+        'callout_placeholder' => 'Geben Sie Ihren Benutzernamen ein, um loszulegen',
+        'callout_button'      => 'Wählen Sie eine Unterschrift',
+
+        'featured' => 'Vorgestellt',
+
+        'about'      => 'Über',
+        'about_text' => [
+            "Ich habe diese Website für die Spieler und Fans des Minecraft-Servers 'Hypixel' erstellt. Es ist der beliebteste Minecraft-Server der Welt, mit vielen öffentlichen Statistiken, mit denen Sie Spaß haben können. Genau das habe ich auf meiner Website getan: Sie können die Minispielstatistiken der Benutzer in Form von dynamischen Signaturen anzeigen: Bilder, die Sie in Forenbeiträgen und in Ihrer Forensignatur platzieren können, damit andere Ihre neuesten Erfolge problemlos angezeigt werden können.",
+            "Der <i> dynamische </ i> Teil bedeutet, dass sich diese Signaturen automatisch aktualisieren. Dies bedeutet, dass Sie Ihre Signatur nur einmal auswählen und verwenden müssen und wir automatisch sicherstellen, dass ihr Inhalt auf dem neuesten Stand ist.",
```
Comment (MaxKorlaar) [inline resources/lang/de/home.php:48]:
```suggestion
            "Der <i>dynamische</i> Teil bedeutet, dass sich diese Signaturen automatisch aktualisieren. Dies bedeutet, dass Sie Ihre Signatur nur einmal auswählen und verwenden müssen und wir automatisch sicherstellen, dass ihr Inhalt auf dem neuesten Stand ist.",
```

File: resources/lang/de/validation.php [hunk 4]
```diff
@@ -0,0 +1,181 @@
...
+        |--------------------------------------------------------------------------
+        |
+        | The following language lines contain the default error messages used by
+        | the validator class. Some of these rules have multiple versions such
+        | as the size rules. Feel free to tweak each of these messages here.
+        |
+        */
+
+        'accepted'             => 'Das :attribute muss akzeptiert werden.',
+        'active_url'           => 'Das :attribute ist keine gültige URL. '
```
Comment (MaxKorlaar) [inline resources/lang/de/validation.php:47]:
```suggestion
        'active_url'           => 'Das :attribute ist keine gültige URL. ',
```

**Review by MaxKorlaar — COMMENTED** [review]
Thank you very much for your work on translating this project and improving these translations as well. I've added a few minor comments and then I believe this PR should be ready!

File: resources/lang/de/guild.php [hunk 5]
```diff
@@ -0,0 +1,150 @@
...
+        ],
+
+        'callout'             => [
+            'Schau dir Informationen zu jeder Hypixel-Gilde an, einschließlich ihrer Mitglieder, Beschreibung und sogar Spielstatistiken.',
+            "Ganz einfach – Sie müssen nur einen Benutzernamen oder einen Gildennamen eingeben, um loszulegen."
+        ],
+        'callout_placeholder' => 'Geben Sie Ihren Benutzernamen ein, um zu starten',
+        'callout_button'      => 'Gilde anzeigen',
+
+        'recently_viewed'                   => 'zuletzt angesehen',
```
Comment (MaxKorlaar) [inline resources/lang/de/guild.php:48]:
```suggestion
        'recently_viewed'                   => 'Zuletzt angesehen',
```

File: resources/lang/de/friends.php (hunk 2 above)
Comment (tintinmaster) [reply resources/lang/de/friends.php:55]: I found out shortly after, after I tried it, but forgot to delete the comment :laughing:

**Review by tintinmaster — COMMENTED** [review]
Looked over it again and it looks fine now.

**Comment by tintinmaster** [comment]
I would look over it again, when it is live on the website, because some things are a bit hard to translate without the context and just the variables

**Review by MaxKorlaar — APPROVED** [review]
Thanks, it looks good! It will be live in a few minutes from now.

**Pull request merged**